package.json

```diff
@@ -80,5 +86,60 @@
       "prettier --write",
       "eslint"
     ]
+  },
+  "standard-version": {
+    "skip": {
+      "tag": true
+    },
+    "types": [
+      {
+        "type": "build",
+        "hidden": true
+      },
+      {
+        "type": "chore",
+        "hidden": true
+      },
+      {
+        "type": "ci",
+        "hidden": true
+      },
+      {
+        "type": "dev",
+        "hidden": true
+      },
+      {
+        "type": "docs",
+        "hidden": true
+      },
+      {
+        "type": "feat",
+        "section": "Features"
+      },
+      {
+        "type": "fix",
+        "section": "Bug Fixes"
+      },
+      {
+        "type": "perf",
+        "hidden": true
+      },
+      {
+        "type": "refactor",
+        "hidden": true
+      },
+      {
+        "type": "revert",
```

Comment:
I'd sort of want this, I could be like "oh, they reverted it in this release" rather than having to wade through all the commit messages/history. And a revert _does_ change the code.